opmon/monitoring.py

```diff
@@ -325,31 +326,28 @@ def _run_sql_for_alerts(self, submission_date) -> None:
             print(f"No alerts configured for {self.normalized_slug}")
             return
 
+        table_name = f"{self.normalized_slug}_alerts_v{SCHEMA_VERSIONS['alert']}"
+        alerts_sql = self._get_sql_for_alerts(submission_date=submission_date)
         try:
-            self.bigquery.client.get_table(
-                f"{self.derived_dataset}.{self.normalized_slug}_alerts_v{SCHEMA_VERSIONS['alert']}"
-            )
-            date_partition = str(submission_date).replace("-", "").split(" ")[0]
-            destination_table = (
-                f"{self.normalized_slug}_alerts_v{SCHEMA_VERSIONS['alert']}${date_partition}"
-            )
-            self.bigquery.execute(
-                self._get_sql_for_alerts(submission_date=submission_date),
-                destination_table,
-                write_disposition=bigquery.job.WriteDisposition.WRITE_TRUNCATE,
-                dataset=f"{self.derived_dataset}",
-            )
+            self.bigquery.client.get_table(f"{self.derived_dataset}.{table_name}")
         except cloud.exceptions.NotFound:
+            # initialize alerts table
             self.bigquery.execute(
-                self._get_metrics_sql(submission_date=submission_date),
-                f"{self.normalized_slug}_alerts_v{SCHEMA_VERSIONS['alert']}",
+                alerts_sql,
+                destination=table_name,
```

Comment:
without this PR, opmon is initializing the alerts table using the metrics sql. is that intended, and should be retained, or was that copypasta?